Doc/library/queue.rst

```diff
@@ -188,9 +188,7 @@ fully processed by daemon consumer threads.
    that had been :meth:`put` into the queue).
 
    Raises a :exc:`ValueError` if called more times than there were items placed in
-   the queue.
-
-   Raises :exc:`ShutDown` if the queue has been shut down immediately.
+   the queue, or after an immediate shutdown.
```

Comment:
That's not entirely correct IIRC. Even after an immediate shutdown, if there are items still being processed, calling `task_done()` for those is allowed. Basically once a `get()` has returned an item, you can call `task_done()` for it (once) any time. Only `get()` and `put()` check the `self.is_shutdown` variable.